common/src/main/java/com/wynntils/screens/settings/widgets/ConfigTile.java

```diff
@@ -86,29 +81,34 @@ private void renderDisplayName(PoseStack poseStack) {
 
     @Override
     public boolean mouseClicked(double mouseX, double mouseY, int button) {
-        double actualMouseX = mouseX - getRenderX();
-        double actualMouseY = mouseY - getRenderY();
+        // Prevent interaction when the tile is outside of the mask from the screen, same applies to drag and released
+        if ((mouseY <= 21 || mouseY >= 205)) return false;
```

Comment:
Can we make this a constant in the screen? It's also hardcoded in other widgets.